src/client/debugger/jupyter/kernelDebugAdapter.ts

```diff
@@ -236,8 +261,15 @@ export class KernelDebugAdapter implements DebugAdapter, IKernelDebugAdapter, ID
         });
     }
 
-    private stackTrace(): void {
-        void this.session.customRequest('stackTrace', { threadId: this.threadId });
+    private stackTrace(args?: {
```

Comment:
Please can we rename this to `getStacktrace`, I saw this earlier and it tripped me, didn't know if it was a Method that got data or a funky method (like a property) that just returned existing existing data or an existing promise